tests/test_secretsmanager/test_secretsmanager.py

```diff
@@ -26,6 +27,19 @@ def test_get_secret_value():
     assert result["SecretString"] == "foosecret"
 
 
+@mock_secretsmanager
+def test_secret_arn():
+    region = "us-west-2"
+    conn = boto3.client("secretsmanager", region_name=region)
+
+    create_dict = conn.create_secret(
+        Name=DEFAULT_SECRET_NAME,
+        SecretString="secret_string",
+    )
+    
```

Comment:
```suggestion

```
Looks like the spaces are still there - maybe your IDE added them back in, in an attempt to be helpful?